src/app/shared/components/about/about.component.ts

```diff
@@ -17,22 +25,47 @@ import { NSSService } from 'app/shared/services/app.service';
 })
 export class AboutModal implements OnInit, OnDestroy {
     @ViewChild('about', {static: true}) public aboutModal; // : ModalDirective;  //modal for validator
+    @ViewChild('f', {static: true}) ticketForm;
     private modalElement: any;
     public CloseResult: any;
     private modalSubscript;
     public appVersion: string;
-
-    constructor(private _nssService: NSSService, private _modalService: NgbModal) {}
+    public Browser: string;
+    public freshDeskTicket : freshDeskTicket;
+    public user: string
+    public token: string
+    public WorkspaceID: string;
+    public RegionID: string;
+    public Server: string;
+    public showSuccessAlert: boolean;
+    public submittingSupportTicket: boolean;
+    public newTicketForm: FormGroup;
+    constructor(
+        private http: HttpClient,
+        public fb: FormBuilder, 
+        private _nssService: NSSService, 
+        public _settingsservice: SettingsService,
+        private _modalService: NgbModal, 
+        private _toasterService: ToasterService,
+        ) {
+        this.newTicketForm = fb.group({
+            'email': new FormControl(null, Validators.required),
+            'subject': new FormControl(null, Validators.required),
+            'description': new FormControl(null, Validators.required),
+            'attachment': new FormControl(null)
+        });
+        this.freshDeskTicket = new freshDeskTicket();
```

Comment:
I think having the form and the interface is unnecessary. I think the `freshDeskTicket` object is just used for the file. Can we merge that with the `newTicketForm`, and access it from there, like the other properties in the `newTicketForm`? Then we can remove that interface file (and imports) too. You should be able to set the value in the uploadFile() function in a similar way that we're clearing the value in the removeFile() function